CMakeLists.txt

```diff
@@ -0,0 +1,31 @@
+project(pystring
+    LANGUAGES CXX
+    )
+
+cmake_minimum_required(VERSION 3.0)
+
+set(CMAKE_CXX_FLAGS "-O3 -Wall -Wextra -Wshadow -Wconversion -Wcast-qual -Wformat=2 ${CMAKE_CXX_FLAGS}")
```

Comment:
This line feels like it will definitely not work for users trying to build with MSVS.

Reply:
That's fine, it was left over from the project I copied the CMakeLists.txt file from as a template. Modify as much as you want :)